Debian Javascript stuff.diff --git a/debian/patches/series b/debian/patches/series
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ef85817..7472d77 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -272,11 +272,11 @@ add_custom_command(
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/glass.png icon_glass
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/source-file.html source_file_text
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/index.html index_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/handlebars.js handlebars_text
+    /usr/share/javascript/handlebars/handlebars.js handlebars_text
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/kcov.js kcov_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/jquery.min.js jquery_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/jquery.tablesorter.min.js tablesorter_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/jquery.tablesorter.widgets.min.js tablesorter_widgets_text
+    /usr/share/javascript/jquery/jquery.min.js jquery_text
+    ${CMAKE_CURRENT_SOURCE_DIR}/../debian/jquery.tablesorter.min.js tablesorter_text
+    ${CMAKE_CURRENT_SOURCE_DIR}/../debian/jquery.tablesorter.widgets.min.js tablesorter_widgets_text
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/tablesorter-theme.css tablesorter_theme_text
    > html-data-files.cc
    DEPENDS
@@ -285,11 +285,11 @@ add_custom_command(
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/glass.png
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/source-file.html
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/index.html
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/handlebars.js
+    /usr/share/javascript/handlebars/handlebars.js
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/kcov.js
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/jquery.min.js
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/jquery.tablesorter.min.js
-    ${CMAKE_CURRENT_SOURCE_DIR}/../data/js/jquery.tablesorter.widgets.min.js
+    /usr/share/javascript/jquery/jquery.min.js
+    ${CMAKE_CURRENT_SOURCE_DIR}/../debian/jquery.tablesorter.min.js
+    ${CMAKE_CURRENT_SOURCE_DIR}/../debian/jquery.tablesorter.widgets.min.js
     ${CMAKE_CURRENT_SOURCE_DIR}/../data/tablesorter-theme.css
     ${CMAKE_CURRENT_SOURCE_DIR}/bin-to-c-source.py
    )
diff --git a/tests/unit-tests/CMakeLists.txt b/tests/unit-tests/CMakeLists.txt
index 50df4be..6254584 100644
--- a/tests/unit-tests/CMakeLists.txt
+++ b/tests/unit-tests/CMakeLists.txt
@@ -51,11 +51,11 @@ add_custom_command(
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/glass.png icon_glass
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/source-file.html source_file_text
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/index.html index_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/handlebars.js handlebars_text
+    /usr/share/javascript/handlebars/handlebars.js handlebars_text
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/kcov.js kcov_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/jquery.min.js jquery_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/jquery.tablesorter.min.js tablesorter_text
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/jquery.tablesorter.widgets.min.js tablesorter_widgets_text
+    /usr/share/javascript/jquery/jquery.min.js jquery_text
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../debian/jquery.tablesorter.min.js tablesorter_text
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../debian/jquery.tablesorter.widgets.min.js tablesorter_widgets_text
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/tablesorter-theme.css tablesorter_theme_text
    > html-data-files.cc
    DEPENDS
@@ -64,11 +64,11 @@ add_custom_command(
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/glass.png
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/source-file.html
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/index.html
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/handlebars.js
+    /usr/share/javascript/handlebars/handlebars.js
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/kcov.js
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/jquery.min.js
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/jquery.tablesorter.min.js
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../data/js/jquery.tablesorter.widgets.min.js
+    /usr/share/javascript/jquery/jquery.min.js
+    ${CMAKE_CURRENT_SOURCE_DIR}/../debian/jquery.tablesorter.min.js
+    ${CMAKE_CURRENT_SOURCE_DIR}/../debian/jquery.tablesorter.widgets.min.js
     ${CMAKE_CURRENT_SOURCE_DIR}/../../data/tablesorter-theme.css
     ${CMAKE_CURRENT_SOURCE_DIR}/../../src/bin-to-c-source.py
    )
